pkg/util/iptables/iptables.go

```diff
@@ -142,12 +143,19 @@ func New(exec utilexec.Interface, dbus utildbus.Interface, protocol Protocol) In
 		glog.Warningf("Error checking iptables version, assuming version at least %s: %v", MinCheckVersion, err)
 		vstring = MinCheckVersion
 	}
+
+	restoreVstring, err := getIPTablesRestoreVersionString(exec)
+	if err != nil {
+		glog.Warningf("Error checking iptables-restore version, assuming version at least %s: %v", MinCheckVersion, err)
```

Comment:
also, in the case of "iptables", we do multiple things with the version string (and also call the get-version-string method from other code), but for iptables-restore we don't so it might be simpler to squash getIPTablesRestoreVersionString into getIPTablesRestoreWaitFlag?